### PR TITLE
Feature/update compute

### PR DESCRIPTION
**Pull request by josepablofm78**
## Description
Adds a complete example using Nevermined SDK about how to run a compute job

### CODE REVIEW

File: docs/architecture/building-blocks.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The complete technical solution includes the following components:
 
 ## Smart Contracts
 
-[Nevermined Smart Contracts](https://github.com/nevermined-io/contracts/)
+[Nevermined Smart Contracts](https://github.com/nevermined-io/contracts/)https://github.com/nevermined-io/docs.githttps://github.com/nevermined-io/docs.gitk
```
Comment (metabacalhau) [inline docs/architecture/building-blocks.md:22]: The text after the link seems off.

**Review by metabacalhau — CHANGES_REQUESTED** [review]

File: docs/architecture/building-blocks.md (hunk 1 above)
Comment (josepablofm78) [reply docs/architecture/building-blocks.md]: fixed!

File: docs/nevermined-sdk/compute-example.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+And the code to publish the asset:
+```ts
+const workflowAttributes = AssetAttributes.getInstance({
+            metadata: workflowMetadata,
+            providers: [config.neverminedNodeAddress]
+        })
+workflowDdo = await nevermined.assets.create(workflowAttributes, publisher)
+```
+
+## Order the Workflow Asset
```
Comment (r-marques) [inline docs/nevermined-sdk/compute-example.md:182]: Here we are ordering the compute service not the workflow

Comment (josepablofm78) [reply docs/nevermined-sdk/compute-example.md]: you are right, fixed!

**Review by metabacalhau — APPROVED** [review]

**Pull request merged**